llvm/cmake/modules/LLVMConfig.cmake.in

```diff
@@ -151,6 +151,9 @@ endif()
 if(NOT TARGET intrinsics_gen)
   add_custom_target(intrinsics_gen)
 endif()
+if(NOT TARGET vt_gen)
```

Comment:
Likewise

Reply:
Do you want me to re-order all of them? `acc_gen` comes after `omp_gen` and `intrinsics_gen` in this list.

Reply:
Ugh, leave this one be I suppose.